core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala

```diff
@@ -203,7 +233,11 @@ class DataCutter(uid: String = UID[DataCutter]) extends Splitter(uid = uid) with
         s" minLabelFraction = $minLabelFract, maxLabelCategories = $maxLabels. \n" +
         s"Label counts were: ${labelCounts.collect().toSeq}")
     }
-    DataCutterSummary(labelsKept.toSeq, labelsDropped.toSeq, labelsDroppedTotal.toLong)
+    DataCutterSummary(
+      labelsKept = labelsKept.toSeq,
```

Comment:
do you want to add the downsample fraction?

Reply:
I didn't want to have to pass the dataset (or the dataset count) to the [estimate](https://github.com/salesforce/TransmogrifAI/blob/213962e4488b43d19fc9c6dc11da1b6869233b12/core/src/main/scala/com/salesforce/op/stages/impl/tuning/DataCutter.scala#L170) function. So I added the dataset count and downsample fraction in the summary variable [here](https://github.com/salesforce/TransmogrifAI/pull/414/files/5aba86cbc9c4bd85551439170daf531d16193afb#diff-b36278615b6efb5334827d4ce645c916L126).